### DISCUSSION TITLE
how can i create an array which can store string or char values??

### DISCUSSION THREAD

**Discussion by ManishAradwad** [modular/modular#1437] [Q&A]
So I'm working on a program which involves creating an array of characters like 

`a = ['1', '2', '3']`

I looked around a lot, asked on the discord, asked the kapa.ai bot but not a single solution seems to be helping.

**Comment by soraros** [comment]
Could you be more specific about what kind of array do you want? Maybe you could just use a string?

**Reply by ManishAradwad** [reply]
So, actually, I want a 2D array that can store character values as elements. Something like

a = [['a', 'b', 'c'],
		['a', 'b', 'c'],
		['a', 'b', 'c']]

The size is known, and in the program, I want to change the values by indexing like

a[0, 2] = 'q'

**Reply by manatlan** [reply]
here is what you want :
https://mojodojo.dev/guides/benchmarks/sudoku.html
